kubemarine/packages.py

```diff
@@ -459,6 +459,25 @@ def _detect_final_package(cluster: KubernetesCluster, detected_packages: Dict[st
         return detected_package_versions[0]
 
 
+def disable_unattended_upgrade(group: NodeGroup) -> None:
+    cluster: KubernetesCluster = group.cluster
+    if group.get_nodes_os() != 'debian':
+        cluster.log.debug("Skipped - unattended upgrades are supported only on Ubuntu/Debian os family")
+        return
+
+    packages_per_node = get_all_managed_packages_for_group(group=group, inventory=cluster.inventory,
+                                                           ensured_association_only=True)
+
+    with group.new_executor() as exe:
+        for node in exe.group.get_ordered_members_list():
+            packages = list(map(lambda package: get_package_name(group.get_nodes_os(), package),
+                                packages_per_node[node.get_host()]))
+            unattended_upgrade_config = 'Unattended-Upgrade::Package-Blacklist { %s };\n' % " ".join(
+                map(lambda package: '"%s";' % package, packages))
```

Comment:
minor style suggestion: one lambda is enough here, or use comprehension
https://google.github.io/styleguide/pyguide.html#27-comprehensions--generator-expressions

Reply:
fixed